content/docs/index.md

```diff
@@ -1,37 +1,36 @@
 # DVC Documentation
 
-**Data Version Control** is a [data versioning], ML [workflow automation], and
-[experiment management] tool that takes advantage of the existing software
-engineering toolset you're already familiar with (Git, your IDE, CI/CD, etc.).
-**DVC** helps data science and machine learning teams manage large datasets,
-make projects reproducible, and better collaborate.
-
-**DVC** is available for [Visual Studio Code], any [system terminal], and as a
-[Python library].
-
-[data versioning]: /doc/start/data-management/data-versioning
-[workflow automation]: /doc/user-guide/pipelines
-[experiment management]: /doc/user-guide/experiment-management
+<admon type="info">
+
+DVC is a tool for data science that takes advantage of existing software
+engineering toolset. It helps machine learning teams manage large datasets, make
+projects reproducible, and collaborate better.
+
+</admon>
+
+**DVC** can be installed on [Visual Studio Code], any [system terminal], and
+used as a [Python library].
```

Comment:
> Difference between this vs. info in https://dvc.org/doc ?

OK. I made this page about the docs more than about DVC now ([similar to TF](https://developer.hashicorp.com/terraform) as well). I still left an admon with a short version of DVC's description.